config/shared/locales.json

```diff
@@ -1,6 +1,12 @@
 [
   { "code": "en", "label": "English" },
-  { "code": "cs", "label": "česky" },
+  { "code": "de", "label": "Deutsch" },
+  { "code": "es", "label": "español, castellano" },
+  { "code": "fr", "label": "français" },
+  { "code": "it", "label": "Italiano" },
+  { "code": "nb", "label": "Norsk bokmål" },
+  { "code": "pl", "label": "polski" },
   { "code": "fi", "label": "suomi" },
```

Comment:
I think we should remove Finnish until there is someone to actually translate it. I'm not gonna have time for it. :-)